src/types/types.ts

```diff
@@ -1 +1,132 @@
 export type Cities = string;
+
```

Comment:
well that's a huge types
Can we maybe move some parts to another type(interface) and reuse to reduce amount of declared fields?